core/includes/common.inc

```diff
@@ -3993,7 +3995,12 @@ function _backdrop_build_css_path($matches, $base = NULL) {
   }
 
   // Prefix with base and remove '../' segments where possible.
-  $path = $_base . $matches[1];
+  if (is_array($matches)) {
+    $path = $_base . $matches[1];
+  }
+  else {
+    $path = $_base . $matches;
+  }
```

Comment:
Looks like `$matches` is either an array or `NULL`. I think this would be better to be more specific and be written as:

```
  if (isset($matches)) {
    $path = $_base . $matches[1];
  }
  else {
    $path = $_base;
  }
```